src/components/common/TmBalance.vue

```diff
@@ -243,6 +266,7 @@ export default {
       rewards: [],
       selectedFiatCurrency: "USD",
       preferredCurrency: "",
+      moonpayCoins: new Set(["ATOM", "LUNA", "SDT", "KAVA"]), // TODO: add warning for US custormers as they cannot purchase any of these coins
```

Comment:
why do we need to do `new Set()`?

Reply:
To know about if they are in the US or not we'd need to ask all our users for their location. They'd get a popup like this:

<img width="340px" src="https://user-images.githubusercontent.com/40721795/81067444-f4d97f80-8ede-11ea-9f19-ccc70a3dc4fc.png">

Or were we getting this info already for GA? 🤔

Reply:
This is done now. Will ask @jrmoreau to confirm it is working